cmake/setup_finalize.cmake

```diff
@@ -84,12 +84,15 @@ ENDFOREACH()
 #
 
 FOREACH(build ${DEAL_II_BUILD_TYPES})
-  CHECK_COMPILER_SETUP(
-    "${DEAL_II_CXX_FLAGS} ${DEAL_II_CXX_FLAGS_${build}}"
-    "${DEAL_II_LINKER_FLAGS} ${DEAL_II_LINKER_FLAGS_${build}}"
-    DEAL_II_HAVE_USABLE_FLAGS_${build}
-    ${DEAL_II_LIBRARIES} ${DEAL_II_LIBRARIES_${build}}
-    )
+# FIXME: until https://github.com/dealii/dealii/issues/3686 is resolved,
+# the simple tests below hangs and renders deal.II unusable.
+  SET(DEAL_II_HAVE_USABLE_FLAGS_${build} TRUE)
+#  CHECK_COMPILER_SETUP(
+#    "${DEAL_II_CXX_FLAGS} ${DEAL_II_CXX_FLAGS_${build}}"
+#    "${DEAL_II_LINKER_FLAGS} ${DEAL_II_LINKER_FLAGS_${build}}"
+#    DEAL_II_HAVE_USABLE_FLAGS_${build}
+#    ${DEAL_II_LIBRARIES} ${DEAL_II_LIBRARIES_${build}}
+#    )
```

Comment:
Yes, I'm uncomfortable removing this entirely. Since you can reproduce the issue, I would rather want us to see what exact command line it is that makes the compiler hangs, and then reduce it to see which flag that is -- we can then selectively do something about that.